regress/pesterTests/FileBasedLogging.tests.ps1

```diff
@@ -0,0 +1,237 @@
+If ($PSVersiontable.PSVersion.Major -le 2) {$PSScriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path}
+Import-Module $PSScriptRoot\CommonUtils.psm1 -Force
+Import-Module OpenSSHUtils -Force
+$tC = 1
+$tI = 0
+$suite = "FileBasedLogging"
+Describe "Tests for admin and non-admin file based logs" -Tags "CI" {
+    BeforeAll {
+        if($OpenSSHTestInfo -eq $null)
+        {
+            Throw "`$OpenSSHTestInfo is null. Please run Set-OpenSSHTestEnvironment to set test environments."
+        }
+        
+        $testDir = "$($OpenSSHTestInfo["TestDataPath"])\$suite"
+        if( -not (Test-path $testDir -PathType Container))
+        {
+            $null = New-Item $testDir -ItemType directory -Force -ErrorAction SilentlyContinue
+        }        
+
+        $sshLogName = "test.txt"
+        $sshdLogName = "sshdlog.txt"
+        $server = $OpenSSHTestInfo["Target"]
+        $opensshbinpath = $OpenSSHTestInfo['OpenSSHBinPath']
+        $nonadminusername = $OpenSSHTestInfo['NonAdminUser']
+        $adminusername = $OpenSSHTestInfo['AdminUser']
+        $password = $OpenSSHTestInfo['TestAccountPW']
+        $port = 47003  		
+        Remove-Item -Path (Join-Path $testDir "*$sshLogName") -Force -ErrorAction SilentlyContinue
+
+        <# Setup sshd_config file#>
+        
+        $sshdconfig_ori = Join-Path $Global:OpenSSHTestInfo["ServiceConfigDir"] sshd_config
+        $sshdconfig_custom = Join-Path $Global:OpenSSHTestInfo["ServiceConfigDir"] sshd_config_custom
+        if (Test-Path $sshdconfig_custom) {
+            Remove-Item $sshdconfig_custom -Force
+        }
+        Copy-Item $sshdconfig_ori $sshdconfig_custom
```

Comment:
You can directly modify the sshd_config file in regress/pestertests/data/ folder.

Reply:
I decided not to do that just because there are other tests that rely on that config file, so I just made a copy of it before I made my changes.